packages/edit-site/src/components/global-styles/screen-style-variations.js

```diff
@@ -78,7 +80,12 @@ function Variation( { variation } ) {
 				onKeyDown={ selectOnEnter }
 				tabIndex="0"
 			>
-				<StylesPreview height={ 100 } />
+				<VStack spacing="2">
+					<div className="edit-site-global-styles-variations_item-preview">
+						<StylesPreview height={ 100 } />
+					</div>
+					<HStack alignment="center">{ variation?.name }</HStack>
```

Comment:
If we go with this approach, we need to add some width 'limits/handling'.
<img width="775" alt="Screenshot 2022-03-14 at 11 43 58 AM" src="https://user-images.githubusercontent.com/16275880/158146783-6a36abcb-f4fe-44d9-b92e-6b775e6eee40.png">